scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -73,7 +73,7 @@ task GetToolVersions {
     # there are a handlful of tasks that require the larger GNU libc-based `slim`.
     String cloud_sdk_slim_docker = "gcr.io/google.com/cloudsdktool/cloud-sdk:435.0.0-slim"
     String variants_docker = "us.gcr.io/broad-dsde-methods/variantstore:2024-02-14-alpine-40124cdc5"
-    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2024_02_16_78c53a6"
+    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2024_03_12_ee216c89aabaf2f85178b40a784a81764148a4c7"
```

Comment:
doesn't need to be fixed for this PR, but the [current version](https://github.com/broadinstitute/gatk/blame/ec490425a54a01dbd708380a8d18c0f4467e1a75/scripts/variantstore/docs/Build%20Docker%20from%20VM/building_gatk_docker_on_a_vm.md#L55) of this script should be producing a shorter hash like the one this replaces

Reply:
Yeah, I usually build the docker locally, using instructions you gave me a while ago. I could change to using the short version of the hash.